src/Controller/AppIcon.php

```diff
@@ -101,11 +101,11 @@ public function getIconData ()
                 $width = is_array($size) ? $size[0] : $size;
                 $height = is_array($size) ? $size[1] : $size;
                 $name = 'icon_' . $config['type'] . '_' . $width . 'x' . $height;
-                $src = 'pub/media/' . self::STORAGE_PATH . $name . '.png';
+                $src = '../' . self::STORAGE_PATH . $name . '.png';
```

Comment:
Can you please leave a comment here regarding why exactly relative path here is needed?

Reply:
manifest.json is located in the /pub/media/webmanifest and it is looking for images inside this directory. I guess we can set absolute path for each image or use relative path as I did, because icons are located in the upper directory.